lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -75,31 +74,6 @@ def inspect
         # @!group Integration steps
         #---------------------------------------------------------------------#
 
-        # Fixes the paths of the copy resource scripts.
-        #
-        # @return [Bool] whether any changes to the project were made.
-        #
-        # @todo   This can be removed for CocoaPods 1.0
-        #
-        def update_to_cocoapods_0_34
```

Comment:
I think we should leave this in, at least until the minimum version of the master specs repository is above 0.34.